app/pages/analyst/organisation-requests.tsx

```diff
@@ -16,21 +16,27 @@ class OrganisationRequests extends Component<Props> {
   static isAccessProtected = true;
   static query = graphql`
     query organisationRequestsQuery(
-      $orderByField: String
-      $direction: String
-      $searchField: String
-      $searchValue: String
+      $user_id: Int
+      $first_name: String
+      $last_name: String
+      $email_address: String
+      $operator_name: String
+      $status: CiipUserOrganisationStatus
+      $order_by: [CiipUserOrganisationsOrderBy!]
     ) {
       query {
         session {
           ...defaultLayout_session
         }
         ...OrganisationRequestsTable_query
           @arguments(
-            orderByField: $orderByField
-            direction: $direction
-            searchField: $searchField
-            searchValue: $searchValue
+            user_id: $user_id
+            first_name: $first_name
+            last_name: $last_name
+            email_address: $email_address
+            operator_name: $operator_name
+            status: $status
+            order_by: $order_by
```

Comment:
A couple things can be refactored in the code below this line:
- remove orderByField & direction from getInitialProps. They are unused. Add a default order_by value if you think it necessary.
- Remove the SearchTable from the render(), just the OrganisationRequestsTable component is needed
- Add pagination (see analyst/applications.tsx for an example)

Reply:
My bad, pagination is added at the ApplicationListContainer level (so OrganisationRequestsTable in this flow)

Reply:
I'm wondering if we should make the pagination part of the `FilterableTableLayout` component instead

Reply:
That's what I was thinking too. We could add a prop to toggle the pagination

Reply:
maybe wait until all of the search user stories are merged, and we can refactor. I am adding some features to FilterableTableLayout in #1523 , so I'd like to avoid conflicts

Reply:
Haha, I was already done ^^ sounds good, it's not much code and worth avoiding tangling things